src/main/java/org/testng/annotations/Ignore.java

```diff
@@ -0,0 +1,20 @@
+package org.testng.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PACKAGE;
+import static java.lang.annotation.ElementType.TYPE;
+
+/**
+ * Alternative of @Test(enable=false)
+ *
+ * Notice that @Ignore on a class will disable all test methods of the class.
+ *
+ */
+@Retention(java.lang.annotation.RetentionPolicy.RUNTIME)
```

Comment:
Can we please add some documentation around how to use this annotation to disable tests especially at the package level ? and also the scope of the annotation when its declared at a package level. If it would disable all tests only in the current package or would it also recursively disable everything in sub-packages as well.

Reply:
I've no idea for sub-packages, I need to test it. Thanks.

Reply:
It didn't work but I fixed it.